templates/scrieDeputatilor.tsx

```diff
@@ -0,0 +1,24 @@
+import generateGoogleMapsLink from '../reusables/generateOSMLink';
+import generateIntroductionDeputati from '../reusables/generateIntroductionDeputati';
+import generateSignature from '../reusables/generateSignature';
+import { getAuthoritiesEmails } from '../reusables/getAuthoritiesEmails';
+import { reportedLocation, userPersonalData } from '../types';
+
+const pistaBicicleteNesigura = {
+  title: 'Pistă de biciclete nesigură',
+  generator: (personal: userPersonalData, location: reportedLocation) => {
+    const introduction = generateIntroductionDeputati(personal);
+    const googleMapsLink = generateGoogleMapsLink(location.lat, location.lng);
+    const signature = generateSignature(personal);
+    return (
+      `${introduction}\n\n` +
+      `{TEXTUL_SCRISORII_AICI}` +
+      `${signature}`
+    );
+  },
+  destination: (localitate: string, judet: string) => {
+    return getAuthoritiesEmails(localitate, judet, 'cameraDeputatilor');
```

Comment:
aici `getAuthoritiesEmails()` o sa incerce sa acceseze sub formatul `${localitate} - ${judet}` din `DEPUTATI`, cred ca mai bine ar fi sa importi `DEPUTATI` in fisierul asta si sa returnezi `DEPUTATI[judet]`

Reply:
de asemenea, poti sa verifici inainte sa returnezi `DEPUTATI['Diaspora']` daca judetul nu se gaseste in obiect, i.e. `if(!(judet in DEPUTATI)) return DEPUTATI['Diaspora']`